draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -474,25 +535,30 @@
         example, a power outage to a pod can cause the Site Physical
         Availability Index to be 0% for all the routes in the pod. Partial
         fiber cut to a row of shelves can cause the Site Physical Availability
-        Index to 50% for all the routes in those shelves. The value is 0-100,
+        Index to be 50% for all the routes in those shelves. The value is 0-100,
         with 100% indicating the site is fully functional, 0% indicating the
         site is entirely out of service, and 50% indicating the site is 50%
         degraded.</t>
 
         <t>It is recommended to assign each route with one Site-ID. Depending
         on deployment, one DC can use POD number as Site-ID, another DC can
         use Row of Shelves as the Site-ID.</t>
+	<!-- XXX JMH - What happens if more than one site id is present?
+	Also, suggest making the above a "for example". -->
```

Comment:
For this version of the document, each route can only be assigned with one Site-ID. 

Added "for example".